### ISSUE TITLE
Publish Vscode-Nim on OpenVSX

### ISSUE THREAD

**Issue by Yu-Vitaqua-fer-Chronos** [nim-lang/vscode-nim#40]
Was asked to make an issue requesting this package to be listed on [OpenVSX](https://open-vsx.org) so it could be kept track of.

I want this since I use Code OSS instead of the official VSC program, and others may do the same.

**Comment by diegovskytl** [comment]
Yep! I am using VSCodium, it would be great to have the extension over there too

**Comment by pierce-jason** [comment]
WORKAROUND: Much thanks to @starkiller on Discord for pointing out that the .vsix file can be downloaded from the Visual Studio extension's web portal by going to the Version History tab.

https://marketplace.visualstudio.com/items?itemName=NimLang.nimlang

Direct download for v0.9.0 .vsix is available at:
https://discord.com/channels/371759389889003530/371759389889003532/1224788753130717214

In VSCodium, enter command mode (ctrl-shift-p), run `ext inst vsix` and browse for the download.

**Comment by Luyten-Orion** [comment]
While yes you can install the VSIX file (which is what I did), it'd be much more convenient to have it published on OpenVSX too

**Comment by SpotlightKid** [comment]
At least attach the vsix file to the tags on Github. Or better, create releases on Github and attach the vsix file there.

**Comment by jmgomez** [comment]
Thanks, from now on we are going to publish the releases to make the process easier. Here is the release for v0.9.0 https://github.com/nim-lang/vscode-nim/releases/tag/v0.9.0

**Comment by Lufen34** [comment]
@jmgomez Thank you ! 😁

**Comment by heheda123123** [comment]
Are there any plans to release to OpenVSX? With vscode completely disabling open source versions of their plugins, cursor windsurf vscodium can't even search for nim-lang plugins anymore!
